be/src/formats/parquet/page_reader.cpp

```diff
@@ -59,6 +60,7 @@ Status PageReader::next_page() {
     if (_opts.use_file_pagecache) {
```

Comment:
Why not set `use_file_pagecache` with `config::parquet_page_cache_skip_over_threshold`?